WalletWasabi/Services/CcjCoordinator.cs

```diff
@@ -131,15 +139,17 @@ public CcjCoordinator(Network network, string folderPath, RPCClient rpc, CcjRoun
 							int newSeverity = found.Value.severity + 1;
 							if(RoundConfig.DosSeverity >= newSeverity)
 							{
-								await UtxoReferee.BanUtxosAsync(newSeverity, found.Value.timeOfBan, prevOut);
+								var txCoins = tx.Outputs.Select(x=> new OutPoint(txId, outputIndex));
+								await UtxoReferee.BanUtxosAsync(newSeverity, found.Value.timeOfBan, txCoins.ToArray());
```

Comment:
It's not good. Neither my own code, nor yours. As you can see I'm indexing through the inputs, so that `outputIndex` is actually `inputIndex` thus you may ban something that doesn't exist.

Reply:
I guess we cannot avoid building the `txCoins` list with a good old for loop.